lab11-amanda/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lab11-amanda",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab 11 single resource express api ======",
+  "main": "index.js",
+  "directories": {
+  "test": "test"
+},
+  "scripts": {
+    "test": "mocha",
+    "start-db":"mkdir -p ./db && mongod --dbpath ./db",
+    "stop-db": "killall mongod",
+    "start": "node index.js"
+  },
+
+  "author": "",
+  "license": "ISC",
+  "directories": {
+    "test": "test"
+  },
+  "dependencies": {
```

Comment:
You are missing `express` from your `dependencies`. You also have a few things in the wrong type of dependency (normal vs dev). You can think of it like this.

What packages would I need to start and run my server. All these go in `dependencies`.
What packages do I need to develop and test my server. All these go in `devDependencies`.